pyomo/contrib/parmest/parmest.py

```diff
@@ -1324,7 +1292,7 @@ def confidence_region_test(
             )
 
         assert isinstance(theta_values, pd.DataFrame)
-        assert distribution in ['Rect', 'MVN', 'KDE']
+        assert distribution in ["Rect", "MVN", "KDE"]
```

Comment:
There are a few `assert` statements here - see earlier comment about `assert` not always being friendly to the user.